	public DockerException(final Throwable cause) {
		super(cause);
	}

